src/Serilog.Enrichers.Thread/project.json

```diff
@@ -1,21 +1,28 @@
 ﻿{
-  "version": "2.0.0-beta-*",
+  "version": "2.0.0-rc-*",
   "description": "Enrich Serilog events with properties from the current thread.",
   "authors": [ "Serilog Contributors" ],
-  "tags": [ "serilog", "thread", "enricher" ],
-  "projectUrl": "http://serilog.net",
-  "licenseUrl": "http://www.apache.org/licenses/LICENSE-2.0",
-  "iconUrl": "http://serilog.net/images/serilog-enricher-nuget.png",
+  "packOptions": {
+    "tags": [ "serilog", "thread", "enricher" ],
+    "projectUrl": "http://serilog.net",
+    "licenseUrl": "http://www.apache.org/licenses/LICENSE-2.0",
+    "iconUrl": "http://serilog.net/images/serilog-enricher-nuget.png"
+  },
   "dependencies": {
-    "Serilog": "2.0.0-beta-511"
+    "Serilog": "2.0.0-rc-577"
   },
-  "compilationOptions": {
-    "keyFile": "../../assets/Serilog.snk"
+  "buildOptions": {
+    "keyFile": "../../assets/Serilog.snk",
+    "xmlDoc": true,
+    "warningsAsErrors": true
   },
   "frameworks": {
-    "net45": {
+    "net4.5": {
```

Comment:
Falls out of the table here: https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md#mapping-the-net-platform-standard-to-platforms